README.md

```diff
@@ -158,9 +158,9 @@ merge(ScopeA, ScopeB) == [
 
 ## 2.5 Delegation
 
-Delegation is the act of granting another principal (the delegate) the capability to use a resource that another has (the delegator). A constructuve "witness" acts as the authorizaton proof for a delegation. Witnesses are either the owning principal's signature or a UCAN with access to that capability in its scope.
+Delegation is the act of granting another principal (the delegate) the capability to use a resource that another has (the delegator). A constructuve "proof" acts as the authorizaton proof for a delegation. Proofs are either the owning principal's signature or a UCAN with access to that capability in its scope.
```

Comment:
Typo (in unchanged word): `constructuve` -> `constructive`

Reply:
Good catch!